qadence_libs/qinfo_tools/qng.py

```diff
@@ -0,0 +1,182 @@
+from __future__ import annotations
+
+from typing import Callable
+
+import torch
+from qadence import QuantumCircuit
+from torch.optim.optimizer import Optimizer, required
+
+from qadence_libs.qinfo_tools.qfi import get_quantum_fisher, get_quantum_fisher_spsa
+
+
+class QNG(Optimizer):
```

Comment:
```suggestion
class QuantumNaturalGradient(Optimizer):
```

People could alias it with `QNG` if they wish to.